FirebaseSegmentation/Tests/Unit/SEGContentManagerTests.m

```diff
@@ -101,7 +122,7 @@ - (void)testAssociateCustomInstallationIdentifierSuccessfully {
 - (FIROptions *)FIRAppOptions {
   FIROptions *options = [[FIROptions alloc] initWithGoogleAppID:@"1:123:ios:123abc"
                                                     GCMSenderID:@"correct_gcm_sender_id"];
-  options.APIKey = @"correct_api_key";
+  options.APIKey = @"AIzaSyabcdefghijklmnopqrstuvwxyz1234567";
```

Comment:
nit: I would make the fake key looking more obviously like a fake one (more repeated laters/numbers, etc. ).

Reply:
I like it! thank you